symbulate/distributions.py

```diff
@@ -342,14 +342,14 @@ class Normal(Distribution):
       sd (float): standard deviation parameter of the normal 
         distribution (if specified, var parameter will be ignored)
     """
+    #TODO edit docstring for Normal Distribution
 
     def __init__(self, mean=0.0, var=1.0, sd=None):
 
         #Note: cleaner way to implement this
 
         if sd is None:
             if (var > 0):
-                self.var = var
```

Comment:
You didn't delete self.var below. In general, when I make a comment, you need to fix the issue everywhere in the code, not just the line where I made the comment.